server/data/nonAssociationsApi.ts

```diff
@@ -1,466 +1,38 @@
-// eslint-disable-next-line max-classes-per-file
+import {
+  NonAssociationsApi as BaseApi,
+  systemUsers as systemUsernames,
+  type NonAssociation,
+  type NonAssociationsList,
+  type OpenNonAssociationsListItem,
+  type ClosedNonAssociationsListItem,
+  type SortBy,
+  type SortDirection,
+} from '@ministryofjustice/hmpps-non-associations-api'
+
+import logger from '../../logger'
 import config from '../config'
 import { nameOfPerson } from '../utils/utils'
 import { transferPrisonId, outsidePrisonId } from './constants'
-import RestClient from './restClient'
+import { restClientMetricsMiddleware } from './restClientMetricsMiddleware'
 import PrisonApi, { type StaffMember } from './prisonApi'
 
-export const roleOptions = {
-  VICTIM: 'Victim',
-  PERPETRATOR: 'Perpetrator',
-  NOT_RELEVANT: 'Not relevant',
-  UNKNOWN: 'Unknown',
-} as const
-export type Role = typeof roleOptions
-
-export const reasonOptions = {
-  BULLYING: 'Bullying',
-  GANG_RELATED: 'Gang related',
-  ORGANISED_CRIME: 'Organised crime',
-  LEGAL_REQUEST: 'Police or legal request',
-  THREAT: 'Threat',
-  VIOLENCE: 'Violence',
-  OTHER: 'Other',
-} as const
-export type Reason = typeof reasonOptions
-
-export const restrictionTypeOptions = {
-  CELL: 'Cell only',
-  LANDING: 'Cell and landing',
-  WING: 'Cell, landing and wing',
-} as const
-export type RestrictionType = typeof restrictionTypeOptions
-
 export const maxCommentLength = 240 as const
```

Comment:
limited comment lengths is not a restriction by the api, but a user interface choice, so it resides here and not in the library